src/web/components/Notification/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import connect from '../../hoc/Color';
```

Comment:
Be careful when using the `connect` name. While we have multiple HOC in the application, it could be great to find other way of using them,

```javascript
colorify(Component)
```

for example

Reply:
Done :smile: